src/services/Config.ts

```diff
@@ -77,6 +84,11 @@ export default class Config {
     this.hostCostPerMonth = setPrice()
     this.disableSelfTest = env.CODIUS_DISABLE_SELF_TEST === 'true'
     this.selfTestSuccess = false
+    this.selfTestConfig = {
+      retryCount: Number(env.CODIUS_SELF_TEST_RETRIES) || 5,
+      retryInterval: Number(env.CODIUS_SELF_TEST_INTERVAL) || 5000
+    }
+    this.selfTestStats = {}
```

Comment:
These stats change dynamically, so they probably shouldn't be a part of the `Config` class.